Adding more detail/examples

ashannon7 opened the pull request:
Updating with a JSON shape strategy based on recent customer challenges and reference data tips.

azurecla commented:
Hi __@v-cmans__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-cmans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ashannon7 committed:
Adding more detail/examples

Updating with a JSON shape strategy based on recent customer challenges and reference data tips.

ashannon7 committed:
Getting JSON to look right

ashannon7 reviewed (commented):
Disregard previous commit.  This is the commit I would like to merge with master.

ashannon7 committed:
Update time-series-insights-send-events.md

ashannon7 reviewed (commented):
One final change.  Sorry for the churn.
